### ISSUE TITLE
Switchmate won't toggle to a different state.

### ISSUE THREAD

**Issue by Wowfunhappy** [valkjsaaa/node-switchmate3#1]
This repository includes discover.switchmate3, polling.switchmate3, and toggle.switchmate3. However, there does not appear to be any sort of pair.switchmate3, or other way to obtain an authentication key.

How do I pair a new switchmate bright with my computer's bluetooth adapter?

**Comment by valkjsaaa** [comment]
The new switchmate is quite different from the old one. The new one do not require to pair and you do not need an authencation key to toggle the switch. (Sounds like some security problems here, but this is the current way it works)

**Comment by Wowfunhappy** [comment]
Thanks for clarifying that! So, I'd originally thought the reason I couldn't toggle my switchmate was because I hadn't paired it, but I guess that's not the case.

```
$ $ ./toggle.switchmate3 90be32cab5c644f4b4ea20369f119b98 on 
Locating 90be32cab5c644f4b4ea20369f119b98.
found
Connected to Switchmate3 90be32cab5c644f4b4ea20369f119b98.
Turning Switchmate3 on.
Attempt failed.
Retrying...
Connected to Switchmate3 90be32cab5c644f4b4ea20369f119b98.
Turning Switchmate3 on.
Attempt failed.
Retrying...
Connected to Switchmate3 90be32cab5c644f4b4ea20369f119b98.
Turning Switchmate3 on.
Attempt failed.
Failed!
```
Interestingly, if my switchmate is already off, and I run the command to toggle it off, it works successfully. The discovery and polling test also worked correctly.

Let me know if I can help debug this in any way.

Edit: While I don't understand most of the code, I decided to try updating the timeout from 5 seconds to 60 seconds. This had no effect.

**Comment by valkjsaaa** [comment]
I found that with the current version of the library, turning off always works while turning on sometimes won't work. I have a fix in mind. Stay tuned for a new version.

**Comment by valkjsaaa** [comment]
I have just published version 0.03. Could you try this new version? Let me know if this resolves the issue.

**Comment by Wowfunhappy** [comment]
Yep, that fixed it! Thank you!
